sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTableCatalogSuite.scala

```diff
@@ -107,6 +111,41 @@ class JDBCTableCatalogSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("simple scan") {
```

Comment:
Could you put the new test to `JDBCTableSuite`. You even didn't touch `JDBCTableCatalog` in this PR.